Formula/grpc.rb

```diff
@@ -80,5 +80,6 @@ def install
     EOS
     system ENV.cc, "test.cpp", "-I#{include}", "-L#{lib}", "-lgrpc", "-o", "test"
     system "./test"
+    system "grpc_cli", "help", "ls"
```

Comment:
Maybe we could even expand this and check some operation of the command? I have no idea what it's supposed to do.

Reply:
It is a CLI to communicate to a service via gRPC. We would need to spin up a separate service which implements gRPC and then try. This was the most minimal test I could think of that reproduces the error.

Reply:
Will it fail with a predictable code and message if you point it to a URL that doesn't run gRPC?

Reply:
This is what happens when pointing it to a non existing instance:
```sh
$ grpc_cli ls localhost:8080
Received an error when querying services endpoint.
ServerReflectionInfo rpc failed. Error code: 14, message: failed to connect to all addresses, debug info: {"created":"@1609938928.336424000","description":"Failed to pick subchannel","file":"/tmp/grpc-20210106-99181-6tbd4s/src/core/ext/filters/client_channel/client_channel.cc","file_line":4166,"referenced_errors":[{"created":"@1609938928.336422000","description":"failed to connect to all addresses","file":"/tmp/grpc-20210106-99181-6tbd4s/src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc","file_line":398,"grpc_status":14}]}

$ echo $?
1
```
We could check for the `grpc_status:14`? According to the docs that's `UNAVAILABLE`: https://github.com/grpc/grpc/blob/master/doc/statuscodes.md